### ISSUE TITLE
When I train the net, loss is nan, What should I do?

### ISSUE THREAD

**Issue by wangxinzhe0617** [sirius-ai/MobileFaceNet_TF#17]
I use the dateset you suggest，witch has 5822653 imges.

**Comment by muyoucun** [comment]
if you use arcface, pretrain the model with softmax firstly
or try to decrease the learning rate

**Comment by wangxinzhe0617** [comment]
@muyoucun thank you very much ,i wil try.

**Comment by sirius-ai** [comment]
minimize the learning rate maybe help for this issue.
close. when necessary reopened.
